bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/GroupItem.java

```diff
@@ -251,7 +255,20 @@ public void stateChanged(Item item, State oldState, State newState) {
      */
     @Override
     public void stateUpdated(Item item, State state) {
+        oldGroupState = this.state; 
         setState(function.calculate(members));
+        sendGroupStateUpdatedEvent(item.getName(), this.state);

```

Comment:
I followed the same approach that is used to notify observers even I don't understand why oberservs are notified if nothing changes. 
See: https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/GenericItem.java#L193